### PR TITLE
update: multus usage example

### PR DESCRIPTION
**Pull request by zcq98**
- [x] Make sure you have followed the [Document Convention](../docs/reference/document-convention.md).

### CODE REVIEW

File: docs/advance/multi-nic.md [hunk 1]
```diff
@@ -70,37 +75,11 @@ spec:
...
+#### 创建一个 Kube-OVN Subnet
 
-- `spec.config.type`: 设置为 `kube-ovn` 来触发 CNI 插件使用 Kube-OVN 子网。
-- `server_socket`: Kube-OVN 通信使用的 socket 文件。 默认位置为 `/run/openvswitch/kube-ovn-daemon.sock`。
-- `provider`: 当前 NetworkAttachmentDefinition 的 `<name>.<namespace>.ovn` , Kube-OVN 将会使用这些信息找到对应的 Subnet 资源，注意后缀需要设置为 ovn。
-
-### 创建一个 Kube-OVN Subnet
-
-创建一个 Kube-OVN Subnet,设置对应的 `cidrBlock` 和 `exclude_ips`, `provider` 应该设置为对应的 NetworkAttachmentDefinition 的 `<name>.<namespace>`,
+创建一个 Kube-OVN Subnet,设置对应的 `cidrBlock` 和 `exclude_ips`，`provider` 应该设置为对应的 NetworkAttachmentDefinition 的 `<name>.<namespace>`，
```
Comment (bobz965) [inline docs/advance/multi-nic.md:82]:
```suggestion
创建一个 Kube-OVN Subnet, 设置对应的 `cidrBlock` 和 `exclude_ips`，`provider` 应该设置为对应的 NetworkAttachmentDefinition 的 `<name>.<namespace>`，
```

File: docs/advance/multi-nic.md [hunk 2]
```diff
@@ -208,3 +169,196 @@ spec:
...
+spec:
+  containers:
+  - name: samplepod-route
+    command: ["/bin/ash", "-c", "trap : TERM INT; sleep infinity & wait"]
+    image: docker.io/library/alpine:edge
+```
+
+##### 创建主网卡为 macvlan 的 Pod
+
+对于使用 macvlan 作为主网卡的 Pod，只需要添加如下 annotation `v1.multus-cni.io/default-network`,取值为对应的 NetworkAttachmentDefinition 的 `<namespace>/<name>`：
```
Comment (bobz965) [inline docs/advance/multi-nic.md:198]:
```suggestion
对于使用 macvlan 作为主网卡的 Pod，只需要添加如下 annotation `v1.multus-cni.io/default-network`, 取值为对应的 NetworkAttachmentDefinition 的 `<namespace>/<name>`：
```

File: docs/advance/multi-nic.md [hunk 3]
```diff
@@ -208,3 +169,196 @@ spec:
...
+  provider: attachnet.default.ovn
+  cidrBlock: 172.17.0.0/16
+  gateway: 172.17.0.1
+  excludeIps:
+  - 172.17.0.0..172.17.0.10
+```
+
+##### 创建一个多网络的 Pod
+
+对于地址随机分配的 Pod，只需要添加如下 annotation `k8s.v1.cni.cncf.io/networks`,取值为对应的 NetworkAttachmentDefinition 的 `<namespace>/<name>`：
```
Comment (bobz965) [inline docs/advance/multi-nic.md:306]:
```suggestion
对于地址随机分配的 Pod，只需要添加如下 annotation `k8s.v1.cni.cncf.io/networks`, 取值为对应的 NetworkAttachmentDefinition 的 `<namespace>/<name>`：
```

File: docs/advance/multi-nic.en.md
Comment (oilbeater) [inline docs/advance/multi-nic.en.md:1]: 这个文件是不是改错了

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: docs/advance/multi-nic.en.md
Comment (zcq98) [reply docs/advance/multi-nic.en.md:1]: 改错了，已修正

**Review by bobz965 — APPROVED** [review]

**Review by oilbeater — APPROVED** [review]

**Pull request merged**